cfgov/unprocessed/css/pages/ask.less

```diff
@@ -328,19 +328,6 @@
                 .respond-to-range( @bp-sm-min, @bp-sm-max, {
                     .h3();
```

Comment:
The heading changes from `h2` to `h3` at smaller sizes, should this be part of the wider pattern? Currently this only happens in the /ask-cfpb/ cards.

Reply:
:+1: that makes sense to me. Changing it from h2 to h3 falls in line with how we typically handle headings at small screen size. I guess Ask CFPB is the only implementation of this as of now? (The only other cards I can think of are the homepage cards, and those don't have headings.)

Reply:
Yes it's ask and the homepage. By default implementing this across the cards would mean the icon on the homepage cards would go from `h2` to `h3` also when the screen size is smaller.

Reply:
oh right— i think that would be fine but would you mind posting a screenshot?

Reply:
h2
<img width="1134" alt="Screen Shot 2020-02-27 at 6 30 29 PM" src="https://user-images.githubusercontent.com/704760/75496487-46213c00-598f-11ea-95e0-c50aef4cf4b6.png">

h3
<img width="833" alt="Screen Shot 2020-02-27 at 6 30 37 PM" src="https://user-images.githubusercontent.com/704760/75496489-47526900-598f-11ea-8cd9-9c8b460c45e0.png">


It's pretty subtle.

Reply:
err sorry, that's a shoddy screenshot because it shows the boxes different sizes. Maybe best if I demo for you.

Reply:
It's fine! When I download the screenshots and look, it's easy to see. I think h3 looks good. Let's go for it! Thanks again for bringing this up.

Reply:
Cool! This responsive sizing is now is https://deploy-preview-428--cfpb-design-system.netlify.com/design-system/components/cards as well